src/index.ts

```diff
@@ -119,11 +121,15 @@ export class Client {
     return this.unary(API.ModelHas, req) as Promise<ModelHasReply.AsObject>
   }
 
-  public async modelFind(storeID: string, modelName: string) {
+  public async modelFind(storeID: string, modelName: string, query: JSONQuery) {
     const req = new ModelFindRequest()
     req.setStoreid(storeID)
     req.setModelname(modelName)
-    return this.unary(API.ModelFind, req) as Promise<ModelFindReply.AsObject>
+    req.setQueryjson(toBase64(JSON.stringify(query)))
+    const res = (await this.unary(API.ModelFind, req)) as ModelFindReply.AsObject
+    // @todo: Do we want to do this? Otherwise, the caller has to decode the base64 string...
```

Comment:
The entities are base64?

Reply:
Yeh, weird right? I didn't think this was the case, but after some digging and trying to 'marshal' straight to JSON, I found they had to be base64 decoded/encoded?!